data/incipias/incipias first contact.txt

```diff
@@ -0,0 +1,338 @@
+			label end
+			`	The presence fades leaving no sign of it ever talking to you.`
+				decline
+
+mission "Incipias: First Contact I"
+	invisible
+	landing
+	to complete
+		never
+	on enter Dom'us
+		conversation
+			`As you enter the system, you notice ships of a type you have never seen before. Hailing them only gives you static. After some more unsuccessful attempts, you give up. Maybe you can find some more information in nearby systems.`
+				to display
+					not "Incipias: Meet the Quarg: offered"
+			`As you enter the system, you notice ships of a type you have never seen before. Hailing them only gives you static. After some more unsuccessful attempts, you give up. This must be the species the Quarg talked about.`
+				to display
+					has "Incipias: Meet the Quarg: offered"
+
+
+
+mission "Incipias: Meet the Quarg"
+	invisible
+	landing
+	source
+		government "Quarg (Incipias)"
+	to complete
+		never
+	on offer
+		conversation
+			branch "ut divitas" "spec inci"
+				has "flagship planet: Ut Divitas"
+			label "ut divitas"
+			`With a soft jolt your spaceship lands. A small group of Quarg quickly gathers around your ship, they make some simple but clear gestures to tell you to wait. After they noticed that you understood them. One of them walks away in a hurrily manner.`
+			`	Only a moment later the Quarg returns with another Quarg. The new one starts talking with a very strange accent.`
+				goto "both planets"
+			label "spec inci"
+			`You land on one of the automised landing pads. As soons as you shut your engines off it starts to move into a hangar slot. The gates of the hangar close. You leave your ship and notice a door on the side of the hangar, a sign there is showing something in a foreign language, but as you get closer it turns white for a moment, before the words "HUMAN DETECTED, PLEASE WAIT WHILE ASSISTANCE IS ON THE WAY" appear. Left with no other choice you have to stay where you are.`
+			`	Luckily it only takes a few minutes until the doors open and three Quarg appear behind it. The one in the middle is wearing clothing that makes clear that it is a military official. The two in the back are wearing full combat gear but do not seem to expect a fight. The one in the middle starts talking in a very strange accent.`
+			label "both planets"
+			`	"Welcome human, long ways came you. Kind of yours never here have been, your mission?"`
+			choice
+				`"I am just exploring the galaxy."`
+					goto "explorer"
+				`"I am looking for new species."`
+					goto "new species"
+			label "explorer"
+			`	"An obvious pursuit for the young that is," it nods, "if advice you I may, to not walk this way in a careless maner, the galaxy holds many dangers that your kind has yet to discover or even to understand." It straightens up to its full height, towering over you, "but in the end worth the trouble it will be."`
```

Comment:
```suggestion
			`	"An obvious pursuit for the young that is," it nods, "if advice you I may, to not walk this way in a careless manner, the galaxy holds many dangers that your kind has yet to discover or even to understand." It straightens up to its full height, towering over you, "but in the end worth the trouble it will be."`
```